monero-outreach-docs/translations/es/merchants/merchant_faq_es.md

```diff
@@ -0,0 +1,58 @@
+# Preguntas frecuentes de los comerciantes de Monero
+
+Cuando arrancamos en Monero se nos vienen muchas preguntas a la cabeza. Hemos contestado algunas de las más comunes y nos gustaría saber que otras podrían haber y responderlas para que otros se beneficien.
```

Comment:
Instead of "nos gustaría saber que otras podrían haber" , I'd change to "nos gustaría saber qué otras podrías tener"